guides/source/action_view_helpers.md

```diff
@@ -620,53 +957,57 @@ link_to "Book", @book # given a composite primary key [:author_id, :id]
 # => <a href="/books/2_1">Book</a>
 ```
 
-You can use a block as well if your link target can't fit in the name parameter. ERB example:
+You can use a block if your link target can't fit in the name parameter.
 
 ```html+erb
 <%= link_to @profile do %>
   <strong><%= @profile.name %></strong> -- <span>Check it out!</span>
 <% end %>
 ```
 
-would output:
+would output the following HTML:
 
 ```html
 <a href="/profiles/1">
   <strong>David</strong> -- <span>Check it out!</span>
 </a>
 ```
 
-See the [API Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to) for more information.
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to)
+for more information.
 
-#### button_to
+#### mail_to
 
-Generates a form that submits to the passed URL. The form has a submit button
-with the value of the `name`.
-
-##### Examples
+Generates a `mailto` link tag to the specified email address. You can also
+specify the link text, additional HTML options, and whether to encode the email
+address.
 
 ```html+erb
-<%= button_to "Sign in", sign_in_path %>
+<%= mail_to "john_doe@gmail.com" %>
 ```
 
-would roughly output something like:
+would output the following HTML:
 
 ```html
-<form method="post" action="/sessions" class="button_to">
-  <input type="submit" value="Sign in" />
-</form>
+<a href="mailto:john_doe@gmail.com">john_doe@gmail.com</a>
 ```
 
-See the [API Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-button_to) for more information.
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-mail_to)
+for more information.
 
-### CsrfHelper
+#### url_for
```

Comment:
Random observation: I see [url_for in the source code](https://github.com/rails/rails/blob/5411787a155e2d1c633a74314edc164d5b300c34/actionview/lib/action_view/helpers/url_helper.rb#L38), but not explicitly documented in the [API helpers](https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html) (although I see it being referenced there)